Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class PlayerController
+{
+    public PlayerModel tankModel { get; private set; }
+    public PlayerView tankView { get; private set; }
+
+    public PlayerController()
```

Comment:
you are always creating the same player controller? 
We should have a unique player controller for each  type of player